src/NuGet.Core/NuGet.Commands/Rules/InvalidPlaceholderFileRule.cs

```diff
@@ -0,0 +1,34 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Globalization;
+using System.IO;
+using NuGet.Packaging;
+
+namespace NuGet.Commands.Rules
+{
+    internal class InvalidPlaceholderFileRule : IPackageRule
+    {
+        private const string PlaceholderFile = "_._";
+
+        public IEnumerable<PackageIssue> Validate(PackageBuilder builder)
+        {
+            foreach (IPackageFile file in builder.Files)
```

Comment:
As we talked about this, this should verify that if a placeholder exists, that it is the only file in the folder.